### DISCUSSION TITLE
Private and project accounts are suspended and did not get reply from GitHub support

### DISCUSSION THREAD

**Discussion by RunzeCui2** [community/community#40331] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My private and project GitHub account are suspended suddenly. This is my first time to use GitHub and I did not notice they did not allow multiple free account for each user. I create a support and hope they unlock my private account and keep suspending the project one. But I they did not reply my email. How can I do to like more effectively get in touch with GitHub supports?

**Comment by RunzeCui2** [comment]
The private account is really important for me since all of my data science homework and notes are uploaded on there. I just hope they unlock my private account. Need helps

**Comment by radar-blip** [comment]
Sometimes it takes a while because they are busy with a lot of things happening. 
Lots of people have questions and other problems that they need help with.

I'm assuming you already tried here, but if you didn't, here you go: 
For account related issues, see here: [Support page](https://support.github.com/contact/account)

**Reply by RunzeCui2** [reply]
Thank you

**Reply by radar-blip** [reply]
No problem. Hang in there!
